# Fundraising / Outreach Executive Summary

As of 2026-04-16, this notebook answers four practical outreach questions in an executive-ready format.

In [ ]:
from pathlib import Path

ANALYSIS_AS_OF_DATE = '2026-04-16'
PROJECT_DIR = Path(r'/Users/liml/Desktop/All Data Zipped/analysis/lapsed_donor_reactivation')
PROJECT_DIR

## Four Answers at a Glance

In [ ]:
# Executive summary table

| question | result | recommended_action |
| --- | --- | --- |
| 1. How many people are reachable by mail but missing an email address? | 16,438 people | Create a dedicated direct-mail, postcard, and email-acquisition segment. |
| 2. Which people added since January are good postcard candidates? | 6 people | Run a small welcome or reactivation postcard test immediately. |
| 3. How large is Leadership Council within the donor base? | 248 people, equal to 3.34% | Treat LC as a distinct stewardship segment, not a proxy for the general donor base. |
| 4. Which lapsed donors should be prioritized for reactivation? | 210 high-priority people, including 210 with a $250+ giving history | Start with Tier 1 and Tier 2, then expand to Tier 3 and Tier 4. |

## 1. How many people are reachable by mail but missing an email address?

The database contains **16,438** people who have a complete mailing address, no email on file, and no `nomail` flag. This is the clearest direct-mail opportunity pool.

![Direct-mail opportunity](./figures/mailing_no_email_opportunity.svg)

In [1]:
mail_reachable_without_email = {
    'total_people': 79830,
    'people_with_complete_mailing_address': 54579,
    'people_with_no_email': 19319,
    'mailing_address_but_no_email_and_mailable': 16438
}
mail_reachable_without_email

{'total_people': 79830, 'people_with_complete_mailing_address': 54579, 'people_with_no_email': 19319, 'mailing_address_but_no_email_and_mailable': 16438}

## 2. Which people added since January are good postcard candidates?

Since **2026-01-01**, the database has added **1,378** people. Only **6** currently qualify for postcard outreach.

![Postcard candidates since January](./figures/people_added_since_january_postcard_opportunity.svg)

In [ ]:
# Postcard candidate list

| vanid | contact_name | city | state_province | date_created | origincodename |
| --- | --- | --- | --- | --- | --- |
| 131942803 | Calderone, Dawna P | Phoenix | AZ | 2026-03-30 | 26Sponsor |
| 131448005 | Melendez, Deanna | Prescott | AZ | 2026-02-26 | 2023 Direct Mail |
| 131448040 | Movahed, Reza | Tucson | AZ | 2026-02-26 |  |
| 130742680 | Bradford Coleman, Karyn | Little Rock | AR | 2026-01-19 |  |
| 130742681 | Figuroa, Lauren | Peoria | AZ | 2026-01-19 |  |
| 130742679 | Jewel, Jasmine | Flagstaff | AZ | 2026-01-19 |  |

## 3. How large is Leadership Council within the donor base?

Leadership Council includes **248** people, and **247** of them are inside the donor base used in this analysis. That equals **3.34%** of person-like donors.

![Donors vs Leadership Council](./figures/donor_vs_leadership_council.svg)

## 4. Which lapsed donors should be prioritized for reactivation?

The analysis identified **353** consistent-but-lapsed donors. Among them, **210** are high-priority and **210** have a `$250+` giving history.

![Lapsed donor priority tiers](./figures/lapsed_donor_priority_tiers.svg)

In [ ]:
# Lapsed donor priority table

| candidate_count | candidate_250plus_count | tier_1_count | tier_2_count | tier_3_count | tier_4_count |
| --- | --- | --- | --- | --- | --- |
| 353 | 210 | 85 | 125 | 54 | 89 |

## Recommended Next Steps

1. Build a dedicated outreach segment for the mail-reachable people who have no email address.
2. Use `./people_added_since_january_postcard_outreach.csv` for an immediate postcard test.
3. Treat Leadership Council as a separate stewardship audience.
4. Start reactivation with Tier 1 and Tier 2 from `./lapsed_consistent_donors_250plus.csv` and `./lapsed_consistent_donors.csv`.